docs/source/ruletypes.rst

```diff
@@ -86,6 +88,20 @@ Rule Configuration Cheat Sheet
 +--------------------------------------------------------------+           |
 | ``kibana_discover_to_timedelta`` (time, default: 10 min)     |           |
 +--------------------------------------------------------------+           |
+| ``shorten_kibana_discover_url`` (boolean, default False)     |           |
```

Comment:
There is shorten_kibana_discover_url, but I added it incorrectly. Please delete it.

Reply:
done